core/src/components/card/test/button/index.html

```diff
@@ -68,6 +68,10 @@
         </ion-card-content>
       </ion-card>
 
+      <ion-card button>
+        <img src="https://images.unsplash.com/photo-1483354483454-4cd359948304?dpr=1&auto=format&fit=crop&w=1000&q=80&cs=tinysrgb&ixid=dW5zcGxhc2guY29tOzs7Ozs%3D">
```

Comment:
We should avoid actual image requests here since end to end testing may or may not have downloaded the image in time for the screenshot, causing our tests to fail often. With many other tests we use a small datauri.

Reply:
@adamdbradley thats fair. I was copying from the existing examples but I can update this in a future PR if you like?

Reply:
@studioromeo I'm going through our tests now to update this, so I'll make sure to update the card test as well 🙂

Reply:
Ok thanks @liamdebeasi !